hono/src/schema/zod-definitions.ts

```diff
@@ -0,0 +1,24 @@
+import { z } from 'zod'
+import { itemTypeEnum } from '../schema/schema'
+
+export const ItemSchema = z.object({
```

Comment:
This definition seems very similar to the `insertItemSchema` on line in `hono/src/services/items.ts`. Maybe there is a way to consolidate these? https://github.com/radicalplatforms/shafa/pull/112/files#diff-a9e37437a7f0a721da7c6f523cfcdef37c5605aaae739da9c365bb881fa795dcR13